smb_description/urdf/sensor_plugins/smb_robosense.gazebo.xacro

```diff
@@ -13,7 +13,7 @@
       <xacro:include filename="$(find robosense_description)/urdf/RS-16.urdf.xacro"/>
 
       <xacro:RS-16 parent="${base_link}" name="rslidar" topic="$(arg pointcloud_topic)" hz="10" samples="440" gpu="${gpu}">
-        <origin xyz="0.00445 0 ${0.298+chassisHeight/2.0}" rpy="0 0 ${M_PI/2.0}" />
+        <origin xyz="0.0245 0 ${0.228+0.1+chassisHeight/2.0}" rpy="0 0 0" />
```

Comment:
would be nice to have parameters for the 0.228 (payload height) and 0.1 (lidar box height).